### PR TITLE
Remove all parts of ProjectApproval StreamForm

### LINKED ISSUE #2583: Remove all parts of Project approval forms model
This is a left over from earlier unfinished development. It will never be used and should be removed.

### PR DESCRIPTION
**Pull request by sks444**
Fixes https://github.com/HyphaApp/hypha/issues/2583

### CODE REVIEW

File: hypha/apply/home/migrations/0002_add_apply_homepage.py [hunk 1]
```diff
@@ -34,6 +34,10 @@ def create_homepage(apps, schema_editor):
 
 class Migration(migrations.Migration):
 
+    run_before = [
+        ('wagtailcore', '0053_locale_model'),  # added for Wagtail 2.11 compatibility
+    ]
+
```
Comment (sks444) [inline hypha/apply/home/migrations/0002_add_apply_homepage.py:40]: This is required after wagtail 2.11.

see: https://docs.wagtail.io/en/stable/releases/2.11.html#run-before-declaration-needed-in-initial-homepage-migration

it was fixed for the [public site](https://github.com/HyphaApp/hypha/blob/9d7fdc3714ef3cea9e04800a6f52a2b35324f53f/hypha/public/home/migrations/0002_create_homepage.py#L40)..but also required for the apply site.

**Comment by sks444** [comment]
@frjo can we deploy this to test just to make sure it doesn't break anything.

**Comment by frjo** [comment]
Deploying to test now.

**Comment by sks444** [comment]
It works well for me.

**Comment by sks444** [comment]
Any idea why the Approving of a submission doesn't create a project on test? I tried on a couple of submissions.

**Comment by frjo** [comment]
@sks444 There is a setting `PROJECTS_AUTO_CREATE` that is set to "False" on test. It is set to "True" in ?dev.py` so you most likely have that on your local setup.

When `PROJECTS_AUTO_CREATE` is set to "False" you get a "Create Project" button in"Actions to take" so staff can do it manually.

**Comment by sks444** [comment]
@frjo Yes, I have `PROJECTS_AUTO_CREATE` set to `True` in my local. Wasn't aware of `Create Project` button, thanks! It works well.

**Review by frjo — APPROVED** [review]
Nice work!

**Pull request merged**